backend/python/app/services/implementations/story_service.py

```diff
@@ -608,8 +604,40 @@ def update_story_translation_stage(self, story_translation_data, user_id):
             self.logger.error(error)
             raise error
 
-    def get_story_translations_available_for_review(self, language, level):
+    def get_stories_available_for_translation(self, language, level, user_id):
         try:
+            ongoing_translations = (
+                self._get_story_translations_user_translating_query(
+                    user_id, isTranslator=True
+                )
+                .filter(StoryTranslation.language == language)
+                .all()
+            )
+            if len(ongoing_translations) > 0:
+                return []
+
+            stories = (
+                Story.query.filter(Story.level == level)
+                .filter(~Story.translated_languages.contains(language))
```

Comment:
just a q: what does the `~Story...` syntax mean?

Reply:
negation! this reads as `return story translations where translated_languages does not contain language`